toolflow/vivado/platform/AU280/AU280.tcl

```diff
@@ -174,6 +174,7 @@ namespace eval platform {
     insert_regslice "host_dma" true "/host/M_DMA" "/memory/S_DMA" "/clocks_and_resets/host_clk" "/clocks_and_resets/host_interconnect_aresetn" ""
     insert_regslice "dma_host" true "/memory/M_HOST" "/host/S_HOST" "/clocks_and_resets/host_clk" "/clocks_and_resets/host_interconnect_aresetn" ""
     insert_regslice "host_arch" true "/host/M_ARCH" "/arch/S_ARCH" "/clocks_and_resets/design_clk" "/clocks_and_resets/design_interconnect_aresetn" ""
+    insert_regslice "l2_cache" [tapasco::is_feature_enabled "Cache"] "/memory/cache_l2_0/M0_AXI" "/memory/mig/C0_DDR4_S_AXI" "/clocks_and_resets/mem_clk" "/clocks_and_resets/mem_peripheral_aresetn" "/memory"
```

Comment:
Is this Regslice necessary? I would assume the "arch_mem" Regslice should be sufficient (if a Regslice is required at all)
Why do you only create it for the first (DDR) cache but not for potential HBM caches?

Reply:
In contrast to the `arch_mem` regslice, this is automatically inserted if the cache feature is enabled. As the port naming changes with the cache, the exisiting regslice cannot be reused.
As far as I remember, timing was not an issue for HBM.